server/tests/auth.test.js

```diff
@@ -43,6 +43,28 @@ describe('AUTH', () => {
         });
     });
 
+    it('should unsuccessfully sign up a user twice', (done) => {
+      chai.request(app)
+        .post('/api/v1/auth/signup')
+        .send({
```

Comment:
@BurnerB , please create a separate folder in your `test` folder and name it `mocks` inside itn create a file called i.e `mocks.js`. inside it you will create instances of data to be used in your test file. It is not ethical to write the data in the test file. rather import the data from another file. please use this in al scenarios including for tokens

Reply:
ok,i'll work on this